examples/custom-service.mdx

```diff
@@ -1,15 +1,33 @@
 ---
 title: "Custom Service"
-description: "This example will explain how to create a custom shuttle service."
+description: "This example will explain how to create a custom shuttle service using Poise and Axum."
 ---
 
+In this simple example we will implement `Service` for a custom service that serves a Discord bot alongside a web server created using Axum.
+
+### Prerequisites
+
+To be able to create this example, we'll need to grab an API token from the [Discord developer portal](https://discord.com/developers/applications).
+
+1. Click the New Application button, name your application and click Create.
+2. Navigate to the Bot tab in the lefthand menu, and add a new bot.
+3. On the bot page click the Reset Token button to reveal your token. Put this token in your `Secrets.toml`. It's very important that you don't reveal your token to anyone, as it can be abused. Create a `.gitignore` file to omit your `Secrets.toml` from version control.
+
+To add the bot to a server, we need to create an invite link:
+
+1. On your bot's application page, open the OAuth2 page via the lefthand panel.
+2. Go to the URL Generator via the lefthand panel, and select the `bot` scope as well as the `Send Messages` permission in the Bot Permissions section.
```

Comment:
This was an oversight as the Discord developer URL generator is somewhat confusing - has been amended

Reply:
Looks like using `application.commands` scope only is correct, I think I got confused trying to use the Discord bot URL gen - this will be fixed in next commit

Reply:
Yeah, I remember writing readmes for serenity and being very confused myself :smile: